extension.json

```diff
@@ -21,6 +21,26 @@
 			"i18n"
 		]
 	},
+	"SpecialPages": {
+		"WikibaseReconcileEdit": {
+			"class": "MediaWiki\\Extension\\WikibaseReconcileEdit\\MediaWiki\\Specials\\SpecialWikibaseReconcileEdit",
+			"factory": "MediaWiki\\Extension\\WikibaseReconcileEdit\\MediaWiki\\Specials\\SpecialWikibaseReconcileEdit::factory"
+		}
+	},
+	"ResourceModules": {
```

Comment:
I assume the JS code can be dropped now that the special page handles form submissions?

Reply:
I was thinking this could be the "rich" experience we give users that have javascript enabled.

Reply:
In that case, I have to point out that `response.html(JSON.stringify(res, null, 2))` is a security vulnerability. Also, we should set up eslint to ensure consistent code style.

I suggest at least deferring this to a separate commit/PR, since the special page should work just fine without it.

Reply:
broke them up into separate patches, this was also rebased on top of T282236.1.